lib/webauthn/public_key_credential_with_assertion.rb

```diff
@@ -16,7 +16,8 @@ def verify(challenge, public_key:, sign_count:, user_verification: nil)
         encoder.decode(challenge),
         public_key: encoder.decode(public_key),
         sign_count: sign_count,
-        user_verification: user_verification
+        user_verification: user_verification,
+        rp_id: client_extension_outputs[:appid] ? appid : nil
```

Comment:
Shouldn't we also need to check the new config is present here?

Reply:
Yes!! That's what we are doing [here](https://github.com/cedarcode/webauthn-ruby/blob/e926b6a1f9a58419c96911be460215eefe60d738/lib/webauthn/public_key_credential_with_assertion.rb#L38:L43) 🙁